adapters/sharethrough/sharethrough.go

```diff
@@ -35,8 +35,13 @@ type SharethroughAdapter struct {
 func (a SharethroughAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters.RequestData, []error) {
 	var reqs []*adapters.RequestData
 
+	if request.Site == nil {
+		return nil, []error{fmt.Errorf("the requests does not have a Site, in App placements are not supported")}
```

Comment:
Minor nit since I have another comment:  it's possible to have better grammar in the above error message.